src/execute/executeService.ts

```diff
@@ -13,7 +13,7 @@ import {
   soapEnv,
   soapBody,
   soapHeader,
-  action
+  action,
```

Comment:
Did we change the prettier/eslint behaviour for trailing commas or did they make a default change?

Reply:
default change

Reply:
@peternhale In our main extensions repo, we changed .prettierrc to have "trailingCommas": false.  Since you said this is a default change in Prettier, do we want to do the manual override here as well?